app/index.tsx

```diff
@@ -7,9 +7,29 @@ import { LedgerBitcoinWallet } from './hooks/useLedgerBitcoinWallet';
 import { LedgerClient } from './ledgerIpc';
 import { ipcRenderer } from 'electron';
 import { createBrowserHistory } from 'history';
+import * as Sentry from '@sentry/react';
+import { captureException, showReportDialog } from '@sentry/react';
 
+Sentry.init({
+  dsn:
+    'https://c4bbd637d7154e338433faf792b7c320@o461938.ingest.sentry.io/5471443'
```

Comment:
Good idea, I will see to give this a try!

Reply:
Additionally I a check so that Sentry is only added in production mode if the `SENTRY_URL` environment variable was present at build time. I also tried to make the menu item conditional, but could not figure that out, so it is only present (but if sentry was not initialized it won't trigger anything).